packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -229,6 +230,15 @@ export function buildRootObject() {
       return zcfMint;
     };
 
+    /** @type TestOnly */
+    const testOnly = testFn => {
```

Comment:
Hmm, can we call this something else? to me, testOnly is test.only which means only one test runs. Could we call this `maketestData` or `addtestContext`? Why take in a function rather than an object directly?

Reply:
I am not attached to the name, and it doesn't particularly need to be short. The reason to take a function is so that the function is not run unless it's in a test context.  So that an actual contract can have a block of code that is just to provide test instrumentation that won't be executed in production. Hence the name is trying to say "this is only run during tests". So a short form of that would be nice. `forTestOnly` `forTestingOnly`? `setTestContext` `ifTestDo`

Reply:
Ooh:  `testJig`